admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/service/helper/ClientSettingsHelper.java

```diff
@@ -65,7 +65,7 @@ private boolean hasURLDetails(Class clazz, boolean isV2API, boolean deltaSync) {
 
 	public Map<Class, CompletableFuture> getInitiateDataFetch(String machineId, String regCenterId,
 			LocalDateTime lastUpdated, LocalDateTime currentTimestamp, boolean isV2API, boolean deltaSync, String fullSyncEntities) {
-		List<String> entities = (fullSyncEntities != null && !fullSyncEntities.isBlank()) ? Arrays.asList(fullSyncEntities.split("\\s*,\\s*")) : new ArrayList<>();
+		List<String> entities = (fullSyncEntities != null && !fullSyncEntities.isBlank()) ? Arrays.asList(fullSyncEntities.split("\\s{0,10},\\s{0,10}")) : new ArrayList<>();
```

Comment:
why is this regex changed?
@HimajaDhanyamraju2  - why can't it just be split on comma? if required lets trim if we are expecting any spaces

Reply:
Updated as suggested